target/linux/ath79/dts/ar9344_tplink_tl-wr841hp-v2.dts

```diff
@@ -0,0 +1,157 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "ar9344.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	model = "TP-Link TL-WR841HP v2";
+	compatible = "tplink,tl-wr841hp-v2", "qca,ar9344";
+
+	aliases {
+		led-boot = &led_system;
+		led-failsafe = &led_system;
+		led-running = &led_system;
+		led-upgrade = &led_system;
+		label-mac-device = &wmac;
+	};
+	leds: leds {
+		compatible = "gpio-leds";
+
+		wlan {
```

Comment:
Please sort leds by something, either name or GPIO number.

Reply:
Sorting is done with respect to GPIO number.